Content.Client/StatusIcon/StatusIconOverlay.cs

```diff
@@ -61,6 +61,9 @@ protected override void Draw(in OverlayDrawArgs args)
             if (icons.Count == 0)
                 continue;
 
+            if (!_statusIcon.IsVisible(uid))
```

Comment:
This visibility check shouldn't apply if the client is an admin observer, otherwise ssd, rev and zombie icons won't be shown to them if the player is hidden in a locker for example.

Reply:
Could ignore the check on all observers more generally too, but admin observer is the most important one.

Reply:
> ssd, rev and zombie icons won't be shown to them if the player is hidden in a locker for example.

i dont see a problem with that

Reply:
![16:19:50](https://github.com/space-wizards/space-station-14/assets/39013340/b2bf2d15-59d1-4683-a723-eb99ffd553ed)

ghost can see invisible mousie